deltachat-ios/DC/DcContext+Extension.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import DcCore
+import WidgetKit
+
+@available(iOS 15, *)
+extension DcContext {
+    private static let key = "ui.ios.selected_apps_for_widget"
+    func shownWidgets() -> [WidgetEntry] {
+        guard let jsonData = getConfig(Self.key)?.data(using: .utf8) else { return [] }
```

Comment:
as discussed in the call, this goes to UserDefaults, but that can be done in a subsequent PR as well, when targeting #2449 

ftr, instead of the function name `shownWidgets()` i would use `getAllWidgetEntries()` and then `storeWidgetEntry()`

Reply:
Thanks for your feedback, adressed it (see below)